engine/apps/api/views/route_regex_debugger.py

```diff
@@ -31,12 +32,28 @@ def get(self, request):
         except re.error:
             raise BadRequest(detail={"regex": ["Invalid regex."]})
 
+        # Previous version did not specify alert_receive_channel and used all of them to search for alerts
+        # Keep this case for backwards compatibility for non-updated plugins
+        if alert_receive_channel_id is None:
+            alert_receive_channels_ids = list(
+                AlertReceiveChannel.objects.filter(
+                    organization_id=organization,
+                    team_id=team,
+                ).values_list("id", flat=True)
+            )
+        else:
+            alert_receive_channels_ids = list(
+                AlertReceiveChannel.objects.get(public_primary_key=alert_receive_channel_id).values_list(
+                    "id", flat=True
+                )
+            )
+
```

Comment:
what about if we made a v2 version of this api endpoint? it would avoid having to have backwards-compatible logic like this within the endpoint

Reply:
I want to add some more thought on how we would start versioning internal api without duplicating the code. Let's merge this for now and discuss how to do that